packages/client/utils/getDraftCoords.ts

```diff
@@ -18,7 +18,7 @@ const getDraftCoords = () => {
   while (target && target !== document) {
     // make sure the selection is inside draft, this isn't always guaranteed
     if ((target as any).className === 'notranslate public-DraftEditor-content') {
-      return getVisibleSelectionRect(window) as FakeClientRect
+      return getVisibleSelectionRect(window) as ClientRect
```

Comment:
It looks like `FakeClientRect` has the same properties as `ClientRect` except the `height` and `width` values are `readonly` in `ClientRect`. Where `getDraftCoords` is being used, the return value is often being used as type `ClientRect` so I think it's best to cast it here. For example, `cachedCoords` is being used [here](https://github.com/ParabolInc/parabol/blob/df50596b028bd866e4800b36cf36f0b034db5521/packages/client/components/TaskEditor/withEmojis.tsx#L95) as the `originCoords` prop which is of type `ClientRect`.